src/plugins/discover/public/application/main/components/layout/discover_documents.tsx

```diff
@@ -277,7 +277,7 @@ function DiscoverDocumentsComponent({
       <>
         <SelectedVSAvailableCallout
           esqlQueryColumns={documents?.esqlQueryColumns}
-          selectedColumns={currentColumns}
+          selectedColumns={currentColumns.filter((col) => col !== '_source')}
```

Comment:
Would you mind adding a comment here explaining why we're filtering out `_source` (similar to the one above)?

Reply:
Sure, thanks for the review!